### ISSUE TITLE
[flutter_webrtc] Support flutter-webrtc

### ISSUE THREAD

**Issue by bwikbs** [flutter-tizen/plugins#291]
There are some requirements for supporting ```webrtc```.

Flutter webrtc project:
https://github.com/flutter-webrtc/flutter-webrtc

Tizen native-api:
https://review.tizen.org/gerrit/#/admin/projects/platform/core/api/webrtc

These could be starting points for webrtc support in flutter-tizen.

**Comment by bbrto21** [comment]
Officially, the webrtc API is supported from Tizen 6.5.
https://docs.tizen.org/application/native/api/mobile/6.5/group__CAPI__MEDIA__WEBRTC__MODULE.html

**Comment by seungsoo47** [comment]
Can you consider using this library as well?
https://github.com/aisouard/libwebrtc

**Comment by bwikbs** [comment]
> https://github.com/aisouard/libwebrtc

IMO, ``flutter-webrtc`` use below :
https://github.com/webrtc-sdk/libwebrtc

So, I would like to quickly making prototype based on this and change it to tizen api later.

**Comment by bwikbs** [comment]
update:

There are two approaches to implementing this plugin. 
The first one is to use the library used in ``flutter-webrtc`` as it is, and the second is to use tizen's api directly. 
When I first tried to use ``https://github.com/webrtc-sdk/libwebrtc`` for quick prototyping, I suffer some build issues. 
Besides, I thought that we should use the Tizen API for various advantages in the end. 
so I decided to use the second way until this moment....
While doing some implementation, I suddenly realized that Tizen api lacked signaling server support. 
This may mean that the plugin uses many external modules for this.(lws, soup, curl whatever..) 
Also, considering the plugin's build environment, this is a situation we want to avoid I guess.

Now I want to go back to the first way. This means that it may take a little longer than I initially expected.
